packages/slice-machine/src/legacy/lib/builders/CustomTypeBuilder/SliceZone/SlicesTemplatesModal.tsx

```diff
@@ -55,11 +55,13 @@ export const SlicesTemplatesModal: FC<UpdateSliceModalProps> = ({
             // Update Redux store
             createSliceSuccess(serverState.libraries);
 
+            const libraryName = localLibraries[0].name;
```

Comment:
same here, `library`, sometimes we use library name, sometimes we use library id... i think when it's a string we can just call it library
```suggestion
            const library = localLibraries[0].name;
```

Reply:
Done ☑️